### PR TITLE
add sounds and dark-mode

### PR DESCRIPTION
**Pull request by Gaurav-Kumar-Soni**
adding sounds and dark mode background.

### CODE REVIEW

File: dino.js [hunk 1]
```diff
@@ -64,6 +65,12 @@ function handleJump(delta) {
 function onJump(e) {
     if (e.code !== "Space" || isJumping) return;
 
+    if (!isJumping) {
+        isJumping = true;
+        yVelocity = JUMP_SPEED;
+        jumpSound.play(); 
+      }
```
Comment (vikram-kangotra) [inline dino.js]: Due to the `isJumping` condition check in the `if` statement above, checking it again is unnecessary. 

remove the condition check on line 68

Also that makes setting the values redundant, so remove the duplicate value set statements.

File: script.js [hunk 2]
```diff
@@ -93,6 +104,23 @@ function updateHighScore() {
...
+function toggleBackground() {
+    if (document.body.classList.contains("dark")) {
+        document.body.classList.remove("dark");
+    } else {
+        document.body.classList.add("dark");
+    }
+}
+
+function updateBackground(){
+    if(Math.floor(score) % 100 == 0 && Math.floor(score) !== prev_score && Math.floor(score) > 0){
```
Comment (vikram-kangotra) [inline script.js]: on line 85, you have used different method of checking for similar conditions.
Be consistent.

Also, increase the time to night.

Comment (Gaurav-Kumar-Soni) [reply script.js]: time to night is increase by 200 multiple of score. is this okay?

File: dino.js [hunk 3]
```diff
@@ -63,7 +64,8 @@ function handleJump(delta) {
 
 function onJump(e) {
     if (e.code !== "Space" || isJumping) return;
-
+        yVelocity = JUMP_SPEED;
+        jumpSound.play(); 
     yVelocity = JUMP_SPEED;
```
Comment (vikram-kangotra) [inline dino.js]: repeated lines (67 and 68)

**Review by vikram-kangotra — APPROVED** [review]

**Comment by Gaurav-Kumar-Soni** [comment]
**Done.**

File: dino.js (hunk 3 above)
Comment (vikram-kangotra) [reply dino.js]: Just fix the spaces before
and then we are done :)

Comment (vikram-kangotra) [reply dino.js]: not the other side 🥲
fix the indentation (left side)

**Pull request merged**